# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-windfarm_rm31.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.7/drivers/macintosh/windfarm_rm31.c.orig	2024-02-18 18:35:38.842532931 +0100
+++ linux-6.7/drivers/macintosh/windfarm_rm31.c	2024-02-18 18:37:51.563520143 +0100
@@ -699,7 +699,7 @@
 	if (nr_chips > NR_CHIPS)
 		nr_chips = NR_CHIPS;
 
-	pr_info("windfarm: Initializing for desktop G5 with %d chips\n",
+	pr_info("windfarm: Initializing for server G5 with %d chips\n",
 		nr_chips);
 
 	/* Get MPU data for each CPU */
